runtime/interpreter/value.go

```diff
@@ -5293,18 +5298,14 @@ func (v *DictionaryValue) GetMember(_ *Interpreter, _ LocationRange, name string
 				keyValue := invocation.Arguments[0]
 				newValue := invocation.Arguments[1]
 
-				existingValue := v.Insert(keyValue, newValue)
+				existingValue := v.Insert(
+					invocation.Interpreter,
+					invocation.LocationRange,
+					keyValue,
+					newValue,
+				)
 
-				var returnValue Value
```

Comment:
this part is no longer needed? just making sure we didn't remove it by accident.

Reply:
This was boxing the result from `Insert`, which used to return a `Value` or `nil`, but is now already returning an `OptionalValue`, so it doesn't need to be boxed up here anymore